cloudformation/media-atom-maker-dev.yml

```diff
@@ -30,6 +34,7 @@ Resources:
     Type: "AWS::IAM::AccessKey"
     Properties:
       UserName: {"Ref": "MediaAtomUser"}
+      Serial: 2
```

Comment:
@adamnfish I agree it's not great and certainly not a pattern to copy anywhere else. I guess we'd need to put the secret key as an output at least otherwise we'd be unable to ever see it?

Happy to chat IRL